Tools/clinic/clinic.py

```diff
@@ -1172,6 +1188,10 @@ def output_templates(
         elif f.kind is GETTER:
             methoddef_define = self.GETTERDEF_PROTOTYPE_DEFINE
             docstring_prototype = docstring_definition = ''
+        elif f.kind is SETTER:
+            return_value_declaration = "int return_value;"
```

Comment:
The return value variable name has a template (`{return_value}`). It _should_ be possible to use it most places, but unfortunately, it is still hard-coded most places 😢 It should map correctly here, though:

```suggestion
            return_value_declaration = "int {return_value};"
```